avro-builder/builder/src/main/java/com/linkedin/avroutil1/builder/SchemaBuilder.java

```diff
@@ -238,6 +244,13 @@ public static void main(String[] args) throws Exception {
       skipCodegenIfSchemaOnClasspath = Boolean.TRUE.equals(Boolean.parseBoolean(value));
     }
 
+    boolean handleUtf8Encoding = true;
+    if (options.has(enableUtf8Encoding)) {
+      String value = options.valueOf(enableUtf8Encoding);
+      handleUtf8Encoding = Boolean.TRUE.equals(Boolean.parseBoolean(value));
+      handleUtf8EncodingInPutByIndex = handleUtf8Encoding;
```

Comment:
This should only be allowed if methodRepresentation and stringRepresentation are charSeq